### PR TITLE
[T4A4][F12-A2] Lee Jun Han Bryan

### PR DESCRIPTION
**Pull request by bryanleejh**
Ready for review

### CODE REVIEW

File: src/seedu/addressbook/data/person/Printable.java [hunk 1]
```diff
@@ -0,0 +1,5 @@
+package seedu.addressbook.data.person;
+
+public interface Printable {
```
Comment (brandonyeoxg) [inline src/seedu/addressbook/data/person/Printable.java:3]: Do remember to add header comments.

**Review by brandonyeoxg — COMMENTED** [review]
@bryanleejh Some comments added. Please close the PR after reading comments.

Recall from tutorial on how we use ISP.
Perhaps you could implement a method to print all Printable objects in another say `ReadOnlyPerson`?

Also I noticed that your lines changed is abnormally high, perhaps you are copy and pasting to recover from a bad code implementation?

File: src/seedu/addressbook/data/person/Printable.java (hunk 1 above)
Comment (bryanleejh) [reply src/seedu/addressbook/data/person/Printable.java:3]: Ah - I think I was trying out the auto formatting thing, and it changed alot of the lines.

**Comment by bryanleejh** [comment]
Hey Brandon, many lines changed cause I was experimenting with the auto formatting in Eclipse. Will be more careful!

**Pull request closed without merging**